be1-go/message/answer.go

```diff
@@ -86,6 +87,17 @@ func (e *Error) Error() string {
 	return e.Description
 }
 
+// Errorf returns an error with the new formatting
+func Errorf(format string, err error) error {
+	msgError := &Error{}
+
+	if xerrors.As(err, &msgError) {
+		msgError.Description = fmt.Sprintf(format, msgError.Description)
+		return msgError
+	}
+	return xerrors.Errorf(format, err)
+}
+
```

Comment:
I created a method which is a little bit similar to `xerrors.Errorf` to be able to add more description to an error of type `Error{}` but I'm not sure it is a good idea. What do you think?